test/test_device_hydro_generation_constructors.jl

```diff
@@ -485,7 +485,7 @@ end
     c_sys5_hy = PSB.build_system(PSITestSystems, "c_sys5_hy")
     model = DecisionModel(MockOperationProblem, DCPPowerModel, c_sys5_hy)
     mock_construct_device!(model, device_model; built_for_recurrent_solves = true)
-    moi_tests(model, true, 48, 0, 48, 24, 0, true)
+    moi_tests(model, true, 48, 0, 72, 48, 0, true)
```

Comment:
@sourabhdalvi these are updates because now there are 2 more constraints for the FF.